src/main/java/calculator/model/Expression.java

```diff
@@ -0,0 +1,34 @@
+package calculator.model;
+
+public class Expression {
+    private static final String EMPTY_SECTION = "";
+    private static final String FIRST_DELIMITER = "//";
+    private static final String LAST_DELIMITER = "\\n";
+
+    private final String expression;
+
+    public Expression(String expression) {
+        this.expression = expression;
+    }
+
+    public boolean hasCustomSeparator() {
+        return expression.startsWith(FIRST_DELIMITER);
+    }
+
+    public String separatorSection() {
+        if (hasCustomSeparator()) {
+            int startIdx = expression.indexOf(FIRST_DELIMITER) + 2;
```

Comment:
저도 비슷한 로직으로 작성해서 처음에 2로 작성했는데 
```suggestion
            int startIdx = expression.indexOf(FIRST_DELIMITER) + FIRST_DELIMITER.length();
```
이런식으로 수정하는게 재사용성 측면에서 좋을 것 같아요!

Reply:
오 이거 너무 좋은 방법이네요! 감사합니다